lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/address/api/service/VerifiableCredentialService.java

```diff
@@ -67,12 +68,27 @@ public Map<String, Object> getAuditEventExtensions(List<CanonicalAddress> addres
                         configurationService.getVerifiableCredentialIssuer(),
                         "VC issuer must not be null"),
                 "addressesEntered",
-                Objects.nonNull(addresses) ? addresses.size() : 0);
+                Objects.nonNull(addresses) ? addresses.size() : 0,
+                "isUkAddress",
+                isUkAddress(getFirstAddressCountryCode(addresses)));
     }
 
     private Object[] convertAddresses(List<CanonicalAddress> addresses) {
         return addresses.stream()
                 .map(address -> objectMapper.convertValue(address, Map.class))
                 .toArray();
     }
+
+    private String getFirstAddressCountryCode(final List<CanonicalAddress> addresses) {
+        if (addresses == null
+                || addresses.get(0) == null
+                || addresses.get(0).getAddressCountry() == null) {
+            return "";
+        }
+        return addresses.get(0).getAddressCountry();
+    }
+
+    private boolean isUkAddress(final String countryCode) {
+        return Stream.of("GB", "GG", "JE", "IM").anyMatch(countryCode::equalsIgnoreCase);
```

Comment:
Can I just check - is there a way that the frontend would currently send GG, JE or JM? My thinking was no, because we send them to the UK address form which hard codes country code GB

Not suggesting we remove this it just plays into another conversation I'm having

Reply:
I have a call about this next week, I _think_ the outcome is going to be that we need to use the correct country codes for crown dependencies but still treat them as UK. 

Let's leave as is for now it's only a small change later if I'm wrong 😅